managed/src/main/java/com/yugabyte/yw/models/Backup.java

```diff
@@ -190,11 +190,10 @@ public static Backup fetchByTaskUUID(UUID taskUUID) {
       .findOne();
   }
 
-  public static List<Backup> getExpiredBackups(UUID scheduleUUID) {
+  public static List<Backup> getExpiredBackups() {
```

Comment:
As mentioned, we should only get expired backups of universes that exist. Otherwise we will keep trying to run delete tasks for backups that cannot be deleted. So after fetching expired backups, you can do a java stream to check all backups if they have universes or not. You can also technically do this in the query itself, but I think a stream to check for existing universe is good enough for now.

Reply:
yeah, missed it. Will update the logic.

Reply:
Updated.

Reply:
@mahendranbhat Fix the getAllUuids function in the Universe class itself. It seems like it isn't referenced from anywhere, and we expect only UUIDs. Also, return a set instead of a list.

Reply:
@Arnav15 thought of fixing, but after seeing the comment above the function, thought it would serve a special purpose and written intentionally. I will refactor it as we are not using it anywhere.